examples/client/Locomotion/src/Components/RidePaymentDetails/index.tsx

```diff
@@ -45,9 +48,17 @@ const RidePaymentDetails = ({
 
   useEffect(() => {
     updatePriceCalculation();
-    loadShowPrice();
   }, []);
 
+  useEffect(() => {
+    if (businessAccountId) {
```

Comment:
lets extract it to a util resusable funcion that gets as params
{ loadShowPrice, getBusinessAccountById, businessAccountId } to reduce code duplication